spark/src/main/scala/geotrellis/spark/io/index/hilbert/HilbertSpaceTimeKeyIndex.scala

```diff
@@ -45,7 +41,7 @@ class HilbertSpaceTimeKeyIndex(
   val timeWidth = keyBounds.maxKey.temporalKey.time.getMillis - startMillis
   val temporalBinCount = math.pow(2, temporalResolution)
  
-  val chc = {
+  lazy val chc = {
```

Comment:
This doesn't quite work. If the lazy val is accessed, I just put a `println(chc)` in the body, then java see will try to follow it (test will fail again with serialization errors). 

It appears that if you make it `@transiet lazy val` we get the behavior we're after.
Relevant SO post: http://stackoverflow.com/questions/27882307/how-does-serialization-of-lazy-fields-work